prover/src/trace/trace_lde/mod.rs

```diff
@@ -49,6 +49,7 @@ pub trait TraceLde<E: FieldElement>: Sync {
         &mut self,
         aux_trace: &ColMatrix<E>,
         domain: &StarkDomain<E::BaseField>,
+        num_partitions: usize,
     ) -> (ColMatrix<E>, <Self::HashFn as Hasher>::Digest);
```

Comment:
As mentioned in one of the previous comments, I think we can get rid of this parameter as we can assume that `TraceLde` keeps track of `num_partitions` since construction.

Reply:
Done